styles/About/About.module.scss

```diff
@@ -0,0 +1,159 @@
+.description {
+  margin: 40px 70px;
+  padding: 30px;
+  box-shadow: var(--primary-shadow);
+  background: var(--primary-white);
+  border-radius: 10px;
+
+  line-height: 1.25;
+  font-size: 1.5em;
+  font-family: var(--acumin-pro);
+  font-weight: 400;
+  font-style: normal;
+
+  p {
+    text-align: center;
+
+    color: var(--dark-grey);
+  }
+}
+
+.team {
+  text-align: center;
+  color: var(--primary-purple);
+
+  margin-top: 50px;
+  margin-bottom: 12px;
+
+  font-size: 3em;
+
+  font-family: var(--expressway);
+  font-weight: 700;
+  font-style: normal;
+}
+
+.profile-container {
+  margin: 45px 200px;
+  margin-bottom: 60px;
+
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(450px, 1fr));
+  grid-gap: 3rem;
+  justify-items: center;
+}
+
+.profile {
+  width: 450px;
+  height: 590px;
+  padding: 30px;
+  border-radius: 10px;
+
+  background-color: var(--primary-white);
+  box-shadow: var(--primary-shadow);
+
+  h3 {
+    color: var(--primary-black);
+
+    text-align: center;
+    margin-top: 8px;
+    margin-bottom: 0px;
+
+    font-size: 2em;
+    font-family: var(--proxima-nova-condensed);
+  }
+
+  p {
+    text-align: left;
+
+    color: var(--dark-grey);
+
+    line-height: 1.3;
+    font-size: 1em;
+    font-family: var(--expressway);
+  }
+}
+
+.profile-pic {
+  margin-left: 25%;
+}
+
+.icon {
+  cursor: pointer;
+  margin-right: 8px;
+}
+
+/* screen must be at least a portrait tablet or small desktop for hover or cursor effects */
+@media (min-width: 940px) {
+  .description p {
+    cursor: default;
+  }
+}
+
+/* most laptop and desktop screens */
+@media (max-width: 1366px) {
+  .description {
+    padding: 25px;
+    box-shadow: var(--mobile-shadow);
+
+    line-height: 1.2;
+    font-size: 1.38em;
+  }
+
+  .description p {
+    margin-top: 12px;
+    margin-bottom: 12px;
+  }
```

Comment:
I'm not sure if we need this? I see that for the general (larger) screen we're not specifying the margins at all so either we can move this to the top or remove it.

Reply:
Oh yes! We can remove. I think I got it from the previous CSS modules and forgot to remove~ 😅

Reply:
Sounds good, just want to make last minute notes before we gather tomorrow on what we need to update! Once we create our mixins we won't have to worry about copy & pasting everywhere either.